src/app/app-routing.module.ts

```diff
@@ -109,7 +109,7 @@ const routes: Routes = [
     canActivate: [AuthVerifiedGuard],
     children: [
       { path: 'dashboard', component: DashboardComponent },
-      { path: 'All_certificate', component: CertificateDetailsComponent },
+      { path: 'All_drec_tokens', component: CertificateDetailsComponent },
```

Comment:
the path should be in kebab case. make sure to use that format for all future route change.

in this case we should just name it to `tokens`